### PR TITLE
fix: add modal confirms when transferring in compare

### PR DESCRIPTION
**Pull request by topliceanurazvan**
## Fixes

- Show a confirm when trying to transfer a resource in the compare & sync
- Show back the extract buttons inside the inspection ( diff )

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/CompareSyncPane/ComparisonList/ComparisonList.tsx [hunk 1]
```diff
@@ -68,13 +70,23 @@ function ComparisonItem({
...
   const handleTransfer = useCallback(
     (side: CompareSide) => {
       const direction = side === 'left' ? 'left-to-right' : 'right-to-left';
-      dispatch(transferResource({ids: [id], direction}));
+      const transferTo = side === 'left' ? rightTypeRef.current : leftTypeRef.current;
+
+      Modal.confirm({
+        title: `Are you sure you want to ${transferTo === 'cluster' ? 'deploy to cluster' : 'extract to local'}?`,
+        onOk() {
+          dispatch(transferResource({ids: [id], direction}));
```
Comment (olensmar) [inline src/components/organisms/CompareSyncPane/ComparisonList/ComparisonList.tsx:84]: telemetry here? include the transferTo and number of resources if possible?

File: src/components/organisms/CompareSyncPane/TransferButton/TransferButton.tsx [hunk 2]
```diff
@@ -43,8 +43,14 @@ const TransferButton: React.FC<Props> = ({side}) => {
   });
 
   const handleTransfer = useCallback(() => {
-    dispatch(transferResource({ids, direction}));
-  }, [direction, dispatch, ids]);
+    Modal.confirm({
+      title: `Are you sure you want to ${buttonLabel.toLowerCase()}?`,
+      onOk() {
+        dispatch(transferResource({ids, direction}));
```
Comment (olensmar) [inline src/components/organisms/CompareSyncPane/TransferButton/TransferButton.tsx:49]: as above..

**Comment by topliceanurazvan** [comment]
We already have telemetry in place for this, it's inside the compare slice, under `transferResource.fulfilled`

image

**Comment by olensmar** [comment]
> 

ok thanks - I think we should strive to do telemetry as close to the user action as possible (and not in slices) so we can see where users are performing actions - but we don't have to change this now - just keep in mind for future telemetry!

**Pull request merged**